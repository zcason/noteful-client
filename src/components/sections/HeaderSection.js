import React, { Component } from 'react';
import { Link, withRouter } from 'react-router-dom';

class HeaderSection extends Component {
    render(){
        return (
            <header>
                <div className="header--spacer"></div>
                <div className="header--title">
                    <Link to={'/'}>
                        <h1>Noteful</h1>
                    </Link>
                </div>
            </header>
       );
    }
}

export default withRouter(HeaderSection);