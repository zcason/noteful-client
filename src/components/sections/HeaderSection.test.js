import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import HeaderSection from './HeaderSection';

describe(`HeaderSection component`, () => {
    it('renders HeaderSection by default', () => {
      const wrapper = shallow(<HeaderSection />)
      expect(toJson(wrapper)).toMatchSnapshot()
    })
  })
  