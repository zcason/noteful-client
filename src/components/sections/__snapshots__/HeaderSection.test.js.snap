// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderSection component renders HeaderSection by default 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
